api/tools/src/main/java/org/pmiops/workbench/tools/ConfigLoader.java

```diff
@@ -69,6 +72,7 @@ public CommandLineRunner run(ConfigDao configDao) {
       String marshalledJson = gson.toJson(configObj, configClass);
       JsonNode marshalledNode = jackson.readTree(marshalledJson);
       JsonNode marshalledDiff = JsonDiff.asJson(newJson, marshalledNode);
+      log.info(marshalledDiff.toString());
```

Comment:
The diff gets logged below as well - was this line for debugging?

Reply:
yes good catch